test/karma.conf.js

```diff
@@ -0,0 +1,107 @@
+// Karma configuration
+
+var fs = require("fs");
+module.exports = function(config) {
+  var customLaunchers = {
+    'SL_Safari_Latest': {
+      base: 'SauceLabs',
+      browserName: 'safari',
+      platform: 'OS X 10.11'
+    },
+    'SL_Safari_8': {
```

Comment:
I haven't got these to run properly yet - only Chrome seems to work on SauceLabs. Will try to get CI working properly and then update the compatibility as appropriate.